### ISSUE TITLE
301 Redirects Not Working - 8.21.2018

### ISSUE THREAD

**Issue by katenolen** [johngodley/redirection#1128]
### Summary

Hi John,

I tried adding 301 redirects to my website yesterday and again this morning, but the redirects are not working.

### Actual Behavior

I am able to set up the redirect without receiving an error message (see attached). But when I click on the old link, it still takes me to the old page with the 404 error (another screenshot is attached).
![301 redirect example](https://user-images.githubusercontent.com/42578601/44406381-7b8ed300-a529-11e8-879e-0e1c793df4f4.jpg) 
![404 error example](https://user-images.githubusercontent.com/42578601/44406446-a37e3680-a529-11e8-96bc-1b0f68bffc97.jpg)

### Environment

I am using Chrome, WordPress version 4.9.6, and Redirection version 3.2.

Thank you,
Katherine Nolen
knolen@bm.net
215.875.3042

**Comment by ragots** [comment]
I'm having the same issues. It does not monitor changes and automatically create a redirect when I change a URL. Basically, It does not do what it's supposed to do.

**Comment by johngodley** [comment]
Did you clear your browser cache? Can you provide a URL to test with?

@ragots that does not sound like the same problem. Please file a separate issue.

**Comment by katenolen** [comment]
Yes, I've cleared my browser cache. Here is an example of a URL I am trying to redirect: https://bergermontague.com/news/2009/vip-chancellor's-award

I want it to redirect to this page: https://bergermontague.com/about-us/honors-awards/

Thanks.

**Comment by ragots** [comment]
Hello, I gave up. 
I purchased a premium plugin that does the job. Too bad because I really like this plugin.

**Comment by johngodley** [comment]
@katenolen you will need to URL encode that quote in the URL - the quote becomes `%27s`
